acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/policy/evaluation/PolicyEvaluationCucumberTest.java

```diff
@@ -36,11 +35,8 @@
 public class PolicyEvaluationCucumberTest extends AbstractTestNGCucumberTests {
     // Used as the entry point for PolicyEvaluation StepsDefinitions
 
-    private ZacTestUtil zacTestUtil = new ZacTestUtil();
-
     @Override
     public void run_cukes() throws JsonParseException, JsonMappingException, IOException {
-        this.zacTestUtil.assumeZacServerAvailable();
```

Comment:
Why are we taking this out? Are we not running cucumber integration tests against deployments without ZAC?